### ISSUE TITLE
CommandBarFlyout secondary command text foreground is not updated after changing from light to dark theme

### ISSUE THREAD

**Issue by winston-de** [microsoft/microsoft-ui-xaml#4970]
**Describe the bug**

The foreground of secondary commands in command bar flyouts doesn't get updated after switching from light to dark theme, and will appear black.

**Steps to reproduce the bug**

1. Open xaml controls gallery with latest winui 2.6 preview and controls resource version set to 2
2. Navigate to CommandBarFlyout page
3. Change system theme from light to dark (or to light then back to dark again)
4. Open CommandBarFlyout and expand
5. Secondary commands have black foreground

**Expected behavior**

The text should change back to white (or whatever the foreground brush is set to) so that the text remains easily visible.

**Screenshots**

![image](https://user-images.githubusercontent.com/59544401/117347510-0d67d700-ae5e-11eb-9e8e-21b77232793e.png)

**Version Info**

NuGet package version: 
Microsoft.UI.Xaml 2.6.0-prerelease.210430001

Windows app type:
| UWP              | Win32            |
| :--------------- | :--------------- |
| Yes |  |

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (19043.964)              | Yes   |
| October 2020 Update (19042)        |    |
| May 2020 Update (19041)            |    |
| November 2019 Update (18363)       |    |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :----------------- | :--------------- |
| Desktop            | Yes |
| Xbox               |  |
| Surface Hub        |  |
| IoT                |  |

**Additional context**

**Comment by StephenLPeters** [comment]
@ranjeshj FYI

**Comment by zhuxb711** [comment]
@StephenLPeters @RBrid @ranjeshj Any progress could share with us?

**Comment by kmgallahan** [comment]
Probably caused by lack of:

```xaml
<Setter Property="Foreground" Value="{ThemeResource CommandBarFlyoutForeground}" />
```

on [`CommandBarFlyoutCommandBarOverflowPresenterStyle`](https://github.com/microsoft/microsoft-ui-xaml/blob/73dbf1d2cdc75316b20332edb582796fb9081d92/dev/CommandBarFlyout/CommandBarFlyout_themeresources.xaml#L606)

**Comment by zhuxb711** [comment]
> Probably caused by lack of:
> 
> ```
> <Setter Property="Foreground" Value="{ThemeResource CommandBarFlyoutForeground}" />
> ```
> 
> on [`CommandBarFlyoutCommandBarOverflowPresenterStyle`](https://github.com/microsoft/microsoft-ui-xaml/blob/73dbf1d2cdc75316b20332edb582796fb9081d92/dev/CommandBarFlyout/CommandBarFlyout_themeresources.xaml#L606)

That solve part of the problem. Now the presenter will follow the theme, but AppBarButton in the SecondaryCommand is still not change its theme.

**Comment by yaira2** [comment]
@ranjeshj fyi

**Comment by yaira2** [comment]
This issue is still occurring, can it be reopened?

**Comment by codendone** [comment]
@yaira2 Have you tried 1.6-preview1? There were some theme-change fixes in 1.6.

**Comment by yaira2** [comment]
I’m currently unable to test due to some crashes in the latest preview. I’ll keep you posted if the issue persists after I resolve those crashes and update.
